src/styles/examples/color.less

```diff
@@ -10,121 +10,166 @@
+        background: @ffe-blue-deep-sky;
+    }
 
-        &--ffe-orange-fire {
-            background: @ffe-orange-fire;
-        }
+    &--ffe-blue-sky {
+        background: @ffe-blue-sky;
+        color: @ffe-black;
+    }
 
-        &--ffe-orange {
-            background: @ffe-orange;
-        }
+    &--ffe-blue-focus {
+        background: @ffe-blue-focus;
+        color: @ffe-black;
+    }
 
-        &--ffe-orange-salmon {
-            background: @ffe-orange-salmon;
-            color: @ffe-black;
-        }
+    &--ffe-blue-pale {
+        background: @ffe-blue-pale;
+        color: @ffe-black;
+    }
 
-        &--ffe-black {
-            background: @ffe-black;
-        }
+    &--ffe-blue-ice {
+        background: @ffe-blue-ice;
+        color: @ffe-black;
+    }
 
-        &--ffe-grey-charcoal {
-            background: @ffe-grey-charcoal;
-        }
+    &--ffe-green-shamrock {
+        background: @ffe-green-shamrock;
+    }
 
-        &--ffe-grey {
-            background: @ffe-grey;
-        }
+    &--ffe-sand {
+        background: @ffe-sand;
+        color: @ffe-black;
+    }
 
-        &--ffe-grey-silver {
-            background: @ffe-grey-silver;
-            color: @ffe-black;
-        }
+    &--ffe-sand-ivory {
+        background: @ffe-sand-ivory;
+        color: @ffe-black;
+    }
 
-        &--ffe-grey-cloud {
-            background: @ffe-grey-cloud;
-            color: @ffe-black;
-        }
+    &--ffe-orange-fire {
+        background: @ffe-orange-fire;
+    }
 
-        &--ffe-white {
-            background: @ffe-white;
-            color: @ffe-black;
-            border: 1px solid @ffe-grey-silver;
-        }
+    &--ffe-orange {
+        background: @ffe-orange;
+    }
 
-        &--ffe-green {
-            background: @ffe-green;
-        }
+    &--ffe-orange-salmon {
+        background: @ffe-orange-salmon;
+        color: @ffe-black;
+    }
 
-        &--ffe-green-emerald {
-            background: @ffe-green-emerald;
-        }
+    &--ffe-black {
+        background: @ffe-black;
+    }
 
-        &--ffe-green-mint {
-            background: @ffe-green-mint;
-            color: @ffe-black;
-        }
+    &--ffe-grey-charcoal {
+        background: @ffe-grey-charcoal;
+    }
 
-        &--ffe-purple {
-            background: @ffe-purple;
-        }
+    &--ffe-grey-warm {
+        background: @ffe-grey-warm;
+        color: @ffe-black;
+    }
 
-        &--ffe-purple-magenta {
-            background: @ffe-purple-magenta;
-        }
+    &--ffe-grey {
+        background: @ffe-grey;
+    }
 
-        &--ffe-red {
-            background: @ffe-red;
-        }
+    &--ffe-grey-silver {
+        background: @ffe-grey-silver;
+        color: @ffe-black;
     }
-}
+
+    &--ffe-grey-cloud {
+        background: @ffe-grey-cloud;
+        color: @ffe-black;
+    }
+
+    &--ffe-white {
+        background: @ffe-white;
+        color: @ffe-black;
+        border: 1px solid @ffe-grey-silver;
```

Comment:
We use `box-sizing: border-box`, so this colors box will be 4 * 1px larger than the others unless they also get a border. I think the border can be `1px solid transparent` though, if you don't want color there.